anaron/Decision_1.3.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE HTML>
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <title> Заголовое страницы</title>
+
+</head>
+
+<body>
+    <h1>HOW-TO: Программа на Python, Часть 2</h1>
+    <ul>
+        <li><em>Программа на Python — часть 1</em></li>
+        <li><em>Программа на Python — часть 2</em></li>
+        <li><em>Программа на Python — часть 3</em></li>
+    </ul>
+    <h2>Содержание</h2>
+    <h3>HOW-TO: Программа на Python, Часть 2</h3>
+    <ul>
+        <li><u>Списки</u></li>
+        <li><u>Комментарии</u></li>
+    </ul>
+    <hr>
+    <p>Впрошлом уроке мы рассмотрели простую программу, которая использовала raw_input
+        для получения ответа пользователя, простые типы переменных и простой цикл «<strong>for</strong>». В
+        этом уроке мы рассмотрим ещё больше типов переменных и напишем ещё больше
+        программ.</p>
+    <h2>Списки</h2>
+    <p>Давайте рассмотрим другой тип переменных, который называется <i>списки</i>. В других
+        языках программирования они называются <i>массивами</i>. Если мы обратимся к аналогии с
+        коробками для обуви, то массив (список) представляет собой несколько склеенных друг с
+        другом коробок с одинаковыми вещами. Например, в одной коробке лежат вилки, в
+        другой — ножи, в третьей — ложки. Посмотрим на простой список— список месяцев. Мы
+        запрограммируем его так:</p>
+    <pre>months =
+['Jan','Feb','Mar','Apr','May','Jun','Jul','Aug','Sep','Oct','Nov','Dec']</pre>
+    <p>Чтобы создать список, мы заключили все значения в квадратные скобки ( '[' и ']' ). Мы
+        назвали наш список '<i>months</i>'. Чтобы использовать список, мы дадим команду print
+        <strong>months[0]</strong> или <strong>months[1]</strong> (которые выведут на экран '<i>Jan</i>' или '<i>Feb</i>'). Помните: счет
+        начинается с нуля. Чтобы найти длину списка, можно использовать такую функцию:</p>
+
+    <code>print len(months)</code>
+
+    <p>которая вернёт 12. </p>
+    <p>Ещё один пример списков — категории в поваренной книге. Например...</p>
+    <code>categories = ['Main dish','Meat','Fish','Soup','Cookies']</code>
+
+    <p>Таким образом, в <strong>categories[0]</strong> хранится <i>'Main dish'</i>, а в <strong>categories[4]</strong> — <i>'Cookies'.</i>. Всё
+        очень просто. Я уверен, вы сможете придумать много других примеров использования
+        списков.</p>
+    <hr>
+    <h2>Комментарии</h2>
+    <p>Далее, мы должны обсудить комментарии. Комментарии важны по нескольким причинам.
+        Они не только говорят вам или другим людям, что вы пытаетесь делать, но и напомнят
+        вам через полгода, что же вы хотели сделать. Когда вы будете писать много программ,
+        комментарии станут важными. Также при их помощи вы заставите Python игнорировать
+        какие-либо части кода. Чтобы закомментировать строку, поставьте перед ней знак '#'.
+        Например:</p>
+    #

```

Comment:
this is a code, need use same tag as you used for code section above

Reply:
i delete this code) because it is old code and I made this hamework last week